src/components/nodes/sequenceFlow/sequenceFlow.vue

```diff
@@ -77,11 +89,23 @@ export default {
         if (newNameLabel !== this.nameLabel) {
           this.nameLabel = newNameLabel;
         }
+        this.defaultFlow = this.isDefaultFlow();
```

Comment:
See comments above regarding the computed property, this could be a call to whatever method handles the stroke at the flow origin